src/applications/financial-status-report/components/FileUploader.jsx

```diff
@@ -78,12 +78,12 @@ const FileUploader = ({ files, onAddFile }) => {
         <strong>Maximum file size: </strong>
       </div>
       <div>50MB</div>
-      <FileInput
-        errorMessage={errorMessage}
-        onChange={handleChange}
-        buttonText="Add files"
-        name="fileUpload"
+      <VaFileInput
         accept={acceptedFileTypes}
+        button-text="Add files"
+        error={errorMessage}
+        name="fileUpload"
+        onVaChange={handleChange}
```

Comment:
Thank you for this migration! I want to mention that the callback function might need to be updated so that it's handling the response as `event?.detail?.files[0]` (instead of just `uploadedFile[0]`). There is an example of this in our Storybook file below:

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/storybook/stories/va-file-input.stories.jsx#L30